js/ui.js

```diff
@@ -284,7 +284,7 @@ function colorizeScripts(msg) {
 		'users'
 	];
 
-	return msg.replace(/(#s.|[^#\.a-z0-9_]|^)([a-z_][a-z0-9_]*)\.([a-z_][a-z0-9_]*)/g, function(match, pre, username, script) {
+	return msg.replace(/(#s.|[^#.a-z0-9_]|^)([a-z_][a-z0-9_]*)\.([a-z_][a-z0-9_]*)/g, function(match, pre, username, script) {
```

Comment:
dot might not need an escape in the character class, but needs an escape here. 
```
#s.
#s\.
```

Reply:
Whoopsie

Reply:
Hah! Absolutely right - that makes a lot more sense. Since it wasn't part of my modifications I totally missed that o.O